docs/use/dapp-staking/for-devs/index.md

```diff
@@ -1,15 +1,98 @@
+import Figure from "/src/components/figure"
+
 # For Developers
 
 ## Overview
 
-Are you a product owner and want to list your dApp in our staking mechanism to earn rewards? You have to make sure you meet the requirements. Read more about those by clicking on the links below.
+Discover [dApp Staking](/docs/learn/dapp-staking/), a unique mechanism that allows communities to support their favorite teams by staking ASTR or SDN with them. When staking ASTR or SDN on a dApp, users not only support the development of innovative apps but also receive staking rewards from inflation.
+
+You can educate yourself more by reading the technical documentation about [dApp Staking](/docs/learn/dapp-staking/).
+
+**Are you a product owner and want to list your dApp in our staking mechanism to earn rewards?**  
+
+Please take the following parameters into consideration and make sure you meet the [requirements](/docs/use/dapp-staking/for-devs/requirements.md):
+
+- To register a dApp for dApp staking you need to have 100 SDN tokens or 1000 ASTR tokens depending on the network you would like to join as a project. dApp staking on our testnet Shibuya requires 100 SBY.
+- **You must claim your rewards to receive them. We recommend that you claim your rewards at the beginning of each new period.**
+
+:::warning
+
+dApp Staking V3 is launched and accessible only on the Shibuya testnet. For dApp owners on Astar and Shiden Network, please refer to the [dApp Staking V2 section](/docs/use/dapp-staking/dapp-staking-v2/for-devs/).
+
+:::
+
+### Periods, Subperiods & Eras:
+
+`Eras`: Time units measured in blocks. Fairly short, a basic time division in dApp staking;
+
+**dApp Staking** is divided into **Periods** which consists of two **Subperiods**: **Voting** and **Build&Earn**.
+
+**Voting** is the subperiod when stakers can decide to vote for dApp(s) to stake their tokens.  
+No staking rewards are generated during the Voting subperiod but if users vote and stake on dApps during this subperiod, they will become eligible for the **Bonus Reward**;
+
+:::important
+
+It's very important for dApp owners and their teams to get organized before and during the voting subperiod to market their products, run campaigns to attract as many stakers and tokens as possible during this period.
+
+:::
+
+**Build&Earn** is the subperiod when stakers and dApps start earning rewards;
+Users can still stake tokens during the Build&Earn subperiod to increase the rewards they get from staking. However, the amount staked during Build&Earn does not contribute to the Bonus Reward.
+
+At the end of a **Build&Earn** subperiod, the current period ends. A new period begins, and all tokens are `unstaked` from dApp(s) but remain `locked`. A new **Voting Subperiod** starts.
```

Comment:
I'd prefer if we don't repeat information, and instead just provide a link to the existing detailed documentation. It's easier to maintain when information isn't duplicated, and it's less error prone.